menoh/CMakeLists.txt

```diff
@@ -59,6 +60,40 @@ if(OpenCV_FOUND)
       "${CHAINER_COMPILER_ROOT_DIR}"
     )
     target_link_libraries(menoh_example PRIVATE menoh_chainer_compiler "${OpenCV_LIBRARIES}")
+    file(
+      DOWNLOAD "https://raw.githubusercontent.com/HoldenCaulfieldRye/caffe/master/data/ilsvrc12/synset_words.txt"
+      "${CMAKE_CURRENT_BINARY_DIR}/synset_words.txt"
+      EXPECTED_HASH SHA256=acf75ef0abe89694b19056e0796401068b459c457baa30335f240c7692857355
+      SHOW_PROGRESS
+      )
+    file(
+      DOWNLOAD "https://upload.wikimedia.org/wikipedia/commons/5/54/Light_sussex_hen.jpg"
+      "${CMAKE_CURRENT_BINARY_DIR}/Light_sussex_hen.jpg"
+      EXPECTED_HASH SHA256=fb13d3a43ef3299d2198a7de7b16e9ee8f0939cd41c33bcd26995c22699ed273
+      SHOW_PROGRESS
+      )
+    # file(
+    #   DOWNLOAD "https://preferredjp.box.com/shared/static/x4k1pya1w1vuvpfk77pp6ma8oaye8syb.onnx"
+    #   "${CMAKE_CURRENT_BINARY_DIR}/resnet50.onnx"
+    #   EXPECTED_HASH SHA256=42916ba7e64f211e1c67aa146e147a01ecd40eaf6b87426dbcd1b7fd9ac8d5d7
+    #   SHOW_PROGRESS
+    #   )
+    file(
+      DOWNLOAD "https://preferredjp.box.com/shared/static/o2xip23e3f0knwc5ve78oderuglkf2wt.onnx"
```

Comment:
It's OK for now, but personally, I don't like VGG for testing since it's too large and too simple. ResNet50 is better but I feel it's still too large. I want this test not run by default at least. I think we can switch to shufflenet once https://github.com/onnx/onnx/pull/2328 is merged (currently, chainer-compiler thinks shufflenet is dynamic shape due to the broken shape inference)